### PR TITLE
TC06 - User is redirected to Home page after logging out

### PR DESCRIPTION
**Pull request by thienan090196**

### CODE REVIEW

File: src/main/java/com/logigear/enums/DriverType.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package com.logigear.enums;
 
 public enum DriverType {
-    CHROME, FIREFOX, EDGE
+    CHROME, FIREFOX
```
Comment (LoliSlayerMC) [inline src/main/java/com/logigear/enums/DriverType.java:4]: why removed?

File: src/main/java/com/logigear/helpers/DriverHelper.java [hunk 2]
```diff
@@ -30,7 +30,7 @@ public static void createDriver(DriverType driverType) {
                 webDriver = new FirefoxDriver();
                 break;
             default:
-                WebDriverManager.chromedriver().setup();
+                WebDriverManager.edgedriver().setup();
```
Comment (LoliSlayerMC) [inline src/main/java/com/logigear/helpers/DriverHelper.java:33]: why changed this?

**Review by LoliSlayerMC — CHANGES_REQUESTED** [review]

File: src/main/java/com/logigear/enums/DriverType.java (hunk 1 above)
Comment (thienan090196) [reply src/main/java/com/logigear/enums/DriverType.java:4]: I don't use that anymore and I set that browser by default when no browser is detected in DriverHelper

File: src/main/java/com/logigear/helpers/DriverHelper.java (hunk 2 above)
Comment (thienan090196) [reply src/main/java/com/logigear/helpers/DriverHelper.java:33]: I replace this to setup Edge Driver when running with Edge

**Review by LoliSlayerMC — APPROVED** [review]
LGTM

**Pull request merged**